### ISSUE TITLE
Nicer print method for results of `eml_validate`

### ISSUE THREAD

**Issue by cboettig** [ropensci/EML#148]
Currently we use the same format as provided by XMLSchemaValidate, which leaves something to be desired.  (On the other hand, some users might be familiar with this format).  Better that the function return `TRUE` (if valid) or `FALSE` along with a helpful `message` or `warning` describing the validation error, if any.

**Comment by amoeba** [comment]
I think returning something with nicer formatting would be great. Returning TRUE/FALSE w/ messages makes sense to me. For the errors, I like one `message()` per error, where each message could have the format:

`{line}.{column}: {msg}`

The above format is similar to what `xmlstarlet val` produces, which is a tool I think some users may be familiar with:

```
xml val -e -s ~/src/eml-trunk/eml.xsd somefile.xml
somefile.xml:153.47: Element 'minimum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:154.47: Element 'maximum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:172.47: Element 'minimum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:173.47: Element 'maximum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:191.47: Element 'minimum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:192.47: Element 'maximum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:210.47: Element 'minimum': '' is not a valid value of the atomic type 'xs:float'.
somefile.xml:211.47: Element 'maximum': '' is not a valid value of the atomic type 'xs:float'.
```

I'll PR this if you think that'd work.

**Comment by cboettig** [comment]
A PR on those lines sounds great.

Currently we just validate the schema via libxml2, ideally we might capture the extra checks done by the online EML validator. That could be done by a form post w rvest or similar package, though that does increase the dependency footprint.  Think that would be worthwhile too?

**Comment by amoeba** [comment]
Hrm, not sure. I do think that it would be nice if `eml_validate` was as fast as reasonably possible to support a use case like validating hundreds of documents in batch. So making an HTTP call for each file might make that use case painful.

By the way, which online EML validator is this? https://knb.ecoinformatics.org/#external//emlparser/ ? Or another one?

**Comment by cboettig** [comment]
Yeah, that one.  Good point, let's stick with the current validation then and just clean up the format

**Comment by cboettig** [comment]
Closed by PR #154
